MantidQt/MantidWidgets/src/DataProcessorUI/DataProcessorGenerateNotebook.cpp

```diff
@@ -49,23 +47,22 @@ specified via the corresponding hinting line edit in the view
 @returns ipython notebook string
 */
 DataProcessorGenerateNotebook::DataProcessorGenerateNotebook(
-    std::string name, const std::string instrument,
+    QString name, const QString instrument,
```

Comment:
Do you know if there is a reason why we don't just have `const QString&` for all cases?